src/modules/qualifications/relations/mentor.js

```diff
@@ -0,0 +1,11 @@
+const { schemaComposer } = require('graphql-compose')
+
+const UserTC = schemaComposer.getOTC('User')
+
+UserTC.addRelation('mentor', {
```

Comment:
You've defined the relationship here and yet no tests was written to prove this works

Reply:
added mentor tests

Reply:
Nope. Still not there. You should check if the mentor is correctly added to a user - to verify this relationship works

Reply:
eg: https://github.com/kyleag/marketplays-api/blob/master/src/tests/departments/team-lead.test.js#L45

Reply:
added